proj.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np # linear algebra
+import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)
+import matplotlib.pyplot as plt
+import seaborn as sns
+import os
+import sqlite3
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
run your parser under `if __name__ == "__main__":`
Read more at: https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/#:~:text=if%20__name__%20%3D%3D%20%E2%80%9Cmain%E2%80%9D%3A%20is%20used%20to,run%20directly%2C%20and%20not%20imported.

Reply:
Added: if __name__ == "__main__": function.